plugin/trino-mariadb/src/test/java/io/trino/plugin/mariadb/TestMariaDbTypeMapping.java

```diff
@@ -876,6 +835,100 @@ public void testTimestampCoercion()
                 .execute(getQueryRunner(), trinoCreateAndInsert("test_timestamp_coercion"));
     }
 
+    @Test
+    public void testDatetime()
+    {
+        testDatetime(UTC);
+        testDatetime(jvmZone);
+        testDatetime(vilnius);
+        testDatetime(kathmandu);
+        testDatetime(TestingSession.DEFAULT_TIME_ZONE_KEY.getZoneId());
+    }
+
+    private void testDatetime(ZoneId sessionZone)
```

Comment:
Please add min and max roundtrips.

Reply:
Added min and max rountrips for `datetime`